.travis.yml

```diff
@@ -0,0 +1,10 @@
+sudo: required
+
+services:
+    - docker
+
+before_install:
+    - docker pull alediaferia/electra
+
+script:
+    - docker run alediaferia/electra /bin/bash -c "su - ecadev; git clone -b $TRAVIS_BRANCH --single-branch https://github.com/Electra-project/Electra.git; cd Electra; qmake; make"
```

Comment:
This is going to be on a Unix machine?

Reply:
It is an Ubuntu-derivate image, yes. I'll look into automating Windows and Mac build soon.